.dockerignore

```diff
@@ -5,9 +5,10 @@
 !pyproject.toml
 !poetry.lock
 !app/
-!shared/
+!dockerfiles/poetry-requirements.txt
```

Comment:
Does the Dependabot config need to be updated to point at this `poetry-requirements.txt`?
If it does, do we want it to be?